### PR TITLE
Fix/readme contribute

### PR DESCRIPTION
**Pull request by LeoTafti**
### Summary

Added a line to the Contribute section to formalize branch naming conventions.

### CODE REVIEW

**Comment by LeoTafti** [comment]
Note: the image in the readme is inserted using the URL of the pic in the GitHub repo (https://github.com/unit8co/darts/raw/develop/static/images/example.png), so the picture in the README won't change until this branch is merged to develop

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ from darts import TimeSeries
 
 df = pd.read_csv('AirPassengers.csv', delimiter=",")
 series = TimeSeries.from_dataframe(df, 'Month', '#Passengers')
-train, val = series.split_after(pd.Timestamp('19590101'))
+train, val = series.split_after(pd.Timestamp('19580101'))
```
Comment (TheMP) [inline README.md:43]: Doesn't seem to be related to this PR

Comment (LeoTafti) [reply README.md:43]: It's not really, but since this PR was small and already about updating the README I thought I could change the example image in the same PR instead of opening a new one…

**Review by hrzn — APPROVED** [review]
